### ISSUE TITLE
microrepositories

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bsorrentino/maven-annotation-plugin#21]
```
I understand that this way of publishing artifact is fairly easy, but it 
introduces a kind of microrepository which just contains this one project.

Users of such microrepositories require either duplicate entries in their 
pom: one for the dependency and one for the repository, or put great burden 
on the maintainer of the local repository to add each microrepository by 
hand. Consider how a pom.xml will look if all dependencies require both a 
repository entry and a dependency entry.

Often companies choose to maintain a locale repository in order not to be 
dependant on the internet and remote sites for their build process. These 
local repositories have the ability to automatically fetch used artifacts 
from remote repositories, but these repositories must be configured first. 
All these microrepositories do not make things easier.

So please instead of this solution have your Maven build post artifacts to 
a real repository like Java's, Maven's or Sontatype's.

Basically what happens is that the artifacts are part of the version 
control system (like CVS, SVN) again and that is exactly one of the things 
we don't do anymore and what Maven was build for.
```

Original issue reported on code.google.com by `tbeer...@gmail.com` on 17 May 2010 at 1:39
